### ISSUE TITLE
Running from source fails: ImportError

### ISSUE THREAD

**Issue by HenryFBP** [twitter-fire-scraper-analytics/twitter-fire-scraper#3]
In `\twitter-fire-scraper\code\twitter-fire-scraper\twitter_fire_scraper\flask_app`, if I run `pipenv run python start_webserver.py`, I get:

```
Traceback (most recent call last):
  File ".\start_webserver.py", line 10, in <module>
    from twitter_fire_scraper.flask_app.forms import ScrapeTermForm
ModuleNotFoundError: No module named 'twitter_fire_scraper'
F:\GitHub\twitter-fire-scraper-analytics\twitter-fire-scraper\code\twitter-fire-scraper\twitter_fire_scraper\flask_app
```

I don't get the same error when running the file from an installed module or in an IDE that sets the system path.

**Comment by HenryFBP** [comment]
Steps to reproduce:

1. `git clone git@github.com:twitter-fire-scraper-analytics/twitter-fire-scraper.git`
2. `cd ./twitter-fire-scraper/code/twitter-fire-scraper/twitter_fire_scraper/`
3. `python3 app.py`
4. ImportError that only happens when running from source

This doesn't happen when running with `pip install twitter-fire-scraper; python3 -m twitter_fire_scraper.app`

**Comment by HenryFBP** [comment]
Fixed from b79d43dc71bfce2d4f632055382e7afdef023f74
